lib/json_schema_form/umm_form_elements/umm_controlled_select.rb

```diff
@@ -0,0 +1,53 @@
+# UmmControlledSelect is used for a select field,
+# but one that has controlled keywords as options
+
+
+# :nodoc:
```

Comment:
We can remove the nodoc since you are adding comments for the class, no?

Reply:
I don't think that the comments I added are really class documentation type comments, so I left them in